docs/Index/baseIndex.md

```diff
@@ -9,7 +9,8 @@
     - [Argumentação](preTraceability/Argumentacao.md)
     - [Plano de Análise de Riscos](preTraceability/PlanAnaliseRiscos.md)
 - **Elicitação**
-    - [StoryBoard](Elicitation/StoryBoard.md)
-    - [Entrevista](Elicitation/Entrevista.md)
-    - [Brainstorming](Elicitation/Brainstorming.md)
-
+    - [StoryBoard](../Elicitation/StoryBoard.md)
```

Comment:
É necessário tirar os "../" pois no localhost ele aparece o documento, mas quando subimos pra master ele não encontra a página. Se deixar só Elicitation/nomedoArquivo.md, quando for para a master ele fará o caminho certo.